test/unit/org/apache/cassandra/service/AutoRepairServiceBasicTest.java

```diff
@@ -67,15 +67,13 @@ public void testsetAutoRepairHistoryClearDeleteHostsBufferInSecV2() {
         assertEquals(100, config.getAutoRepairHistoryClearDeleteHostsBufferInterval().toSeconds());
     }
 
-
```

Comment:
While at it, can you fix the braces on this and `AutoRepairServiceRepairTypeTest`?

Reply:
Done